docs/contributor_guide/development_process.rst

```diff
@@ -80,6 +82,11 @@ To verify that the code works as expected run:
    $ python ./scripts/install_requirements.py --pinned False
    $ python -m pytest -s ./test/unit
 
+.. note::
+
+   If there is a :code:`torch` related error during the installation,
+   please downgrade keep your :code:`python` version between 3.8-3.12.
```

Comment:
This will need updating when the PR to drop 3.8 and add 3.13 merges...

Reply:
thanks for the reminder! we are not ready to encourage the 3.13 usage, so it has to be 3.9-3.12 at the moment. added in https://github.com/fairlearn/fairlearn/pull/1486/commits/33c10615ef1fc309607a1cc42088db7c9d78d86b